assets/js/app.ts

```diff
@@ -147,41 +147,58 @@ const Hooks = {
     },
   },
   VideoPlayer: {
-    mounted() {
+    async mounted() {
       const backdrop = document.querySelector("#video-backdrop");
-
       this.playerId = this.el.id;
 
       // TODO: remove this once we have a better way to handle autoplay
       const autoplay = this.el.id.startsWith("analytics-") ? false : "any";
 
-      this.player = videojs(this.el, {
-        autoplay: autoplay,
-        liveui: true,
-        html5: {
-          vhs: {
-            llhls: true,
-          },
-        },
+      this.player = await window.VidstackPlayer.create({
+        target: this.el,
+        autoplay,
+        viewType: "video",
+        streamType: "on-demand",
+        logLevel: "warn",
+        crossOrigin: true,
+        playsInline: true,
+        live: true,
+        seeking: true,
+        poster:
```

Comment:
Let's set this inside `playVideo` using a new `poster` param

Reply:
oh okay. I'll update this now.